pkg/network/network.go

```diff
@@ -136,6 +140,75 @@ func Render(conf *opv1alpha1.NetworkAddonsConfigSpec, manifestDir string, opensh
 	return objs, nil
 }
 
+// RenderObjsToRemove creates removal component object list
+func RenderObjsToRemove(prev, conf *opv1alpha1.NetworkAddonsConfigSpec, manifestDir string, openshiftNetworkConfig *osv1.Network, clusterInfo *ClusterInfo) ([]*unstructured.Unstructured, error) {
+	log.Print("starting rendering objects to delete phase")
+	objsToRemove := []*unstructured.Unstructured{}
+	o := []*unstructured.Unstructured{}
+	var err error
+
+	operatorNamespace := os.Getenv("OPERATOR_NAMESPACE")
+
+	if conf.Multus == nil && prev != nil {
+		// render Multus
+		o, err = renderMultus(prev, manifestDir, openshiftNetworkConfig, clusterInfo)
+		if err != nil {
+			return nil, err
+		}
+		objsToRemove = append(objsToRemove, o...)
+	}
+
+	if conf.LinuxBridge == nil && prev != nil {
+		// render LinuxBridge for Removal
+		o, err = renderLinuxBridge(prev, manifestDir, clusterInfo)
+		if err != nil {
+			return nil, err
+		}
+		objsToRemove = append(objsToRemove, o...)
+	}
+
+	if conf.KubeMacPool == nil && prev != nil {
+		// render kubeMacPool
+		o, err = renderKubeMacPool(prev, manifestDir)
+		if err != nil {
+			return nil, err
+		}
+		objsToRemove = append(objsToRemove, o...)
+	}
+
+	if conf.NMState == nil && prev != nil {
+		// render NMState for Removal
+		o, err = renderNMState(prev, manifestDir, clusterInfo)
+		if err != nil {
+			return nil, err
+		}
+		objsToRemove = append(objsToRemove, o...)
+	}
+
+	if conf.Ovs == nil && prev != nil {
+		// render Ovs for Removal
+		o, err = renderOvs(prev, manifestDir, clusterInfo)
+		if err != nil {
+			return nil, err
+		}
+		objsToRemove = append(objsToRemove, o...)
+	}
+
+	// Remove Namespace occurences, since its not allowed to remove it, as cnao pod is there
```

Comment:
Just filter out all Namespace type objects. Also, you need to read OPERAND_NAMESPACE if anything.

Reply:
you mean to change the comment ? 
i do read OPERATOR_NAMESPACE at func begining
or need also OPERAND_NAMESPACE ?

Reply:
do we take this in the future on other PR ?
or what is the AI here please?